readthedocs/doc_builder/python_environments.py

```diff
@@ -40,138 +37,6 @@ def __init__(self, version, build_env, config=None):
             version_slug=self.version.slug,
         )
 
-    def install_build_tools(self):
```

Comment:
I moved this because it's agnostic to the language. It could install only NodeJS. So, it doesn't make sense to be on `PythonEnvironment` class. I needed to call `install_build_tools` _without_ creating an environment when using `build.commands`.